src/main/policy.cc

```diff
@@ -59,9 +60,6 @@ int infopolicy_from_jsobject(as_policy_info* policy, Local<Object> obj, const Lo
 	if ((rc = get_optional_uint32_property(&policy->timeout, NULL, obj, "timeout", log)) != AS_NODE_PARAM_OK) {
 		return rc;
 	}
-	if ((rc = get_optional_uint32_property(&policy->timeout, NULL, obj, "totalTimeout", log)) != AS_NODE_PARAM_OK) {
-		return rc;
-	}
```

Comment:
The `timeout` property is used here just as a fallback to remain backwards compatibility. As mentioned earlier, I think we should stick to using the same name `totalTimeout` as the other (C/Java) clients. We should removed the fallback to also check the `timeout` property from the 4.0 release.

Reply:
infoPolicy has timeout and basePolicy has total_timeout fields, so you mean to say use corresponding names "timeout" and "totalTimeout" from user objects?

Reply:
Yes. The base policy has two timeout values, i.e. `totalTimeout` and `socketTimeout`. Just using `timeout` might be a bit ambiguous.